match.py

```diff
@@ -9,6 +9,7 @@
 c = Configuration("config.json")
 client = MongoClient(c.mongo_uri())
```

Comment:
Drop parens here, too: should be `client = MongoClient(c.mongo_uri)` to match property.